frontend/.env.development

```diff
@@ -9,8 +9,9 @@ NEXT_PUBLIC_MATOMO_URL=https://matomo.ubreakit.com
 NEXT_PUBLIC_GA_KEY=UA-30506-1
 # If we want to test Google Analytics in dev DEBUG should be true and URL should be
 # https://www.google-analytics.com/analytics_debug.js
-NEXT_PUBLIC_GA_DEBUG=false
+NEXT_PUBLIC_GA_DEBUG=true
 NEXT_PUBLIC_GA_URL=
+NEXT_PUBLIC_GTAG_ID=G-5ZXNWJ73GK
```

Comment:
If this will be the same everywhere, we can put in the `.env` file (we can create it if it doesn't exist yet)

https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#default-environment-variables